shell.nix

```diff
@@ -0,0 +1,52 @@
+{ pkgs ? import <nixpkgs> {} }:
```

Comment:
I would prefer this is added in a separate PR (and you would agree to take on responsibility for maintaining it)

Reply:
That's fine!